### PR TITLE
Update pipelines workload

### PR DESCRIPTION
**Pull request by wkulhanek**

### CODE REVIEW

File: ansible/roles/ocp4-workload-pipelines/tasks/remove_workload.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  - ./templates/subscription.j2
+
+- name: Remove openshift-pipelines project
+  k8s:
+    state: absent
+    api_version: project.openshift.io/v1
+    kind: Project
+    name: openshift-pipelines
+
+- name: Remove tkn and tkn bash completion
```
Comment (stencell) [inline ansible/roles/ocp4-workload-pipelines/tasks/remove_workload.yml]: Not that it is hurting, but is this necessary to remove?

Comment (wkulhanek) [reply ansible/roles/ocp4-workload-pipelines/tasks/remove_workload.yml]: Well. The workload installs it. So I feel removing the workload should remove it.

Comment (wkulhanek) [reply ansible/roles/ocp4-workload-pipelines/tasks/remove_workload.yml]: Hm. Maybe CRDs need cleaning up, too...

File: ansible/roles/ocp4-workload-pipelines/tasks/remove_workload.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    api_version: operators.coreos.com/v1alpha1
+    kind: Subscription
+    name: openshift-pipelines-operator
+    namespace: openshift-operators
+  register: r_subscription
+
+- name: Print CSV
+  debug:
+    msg: "CSV: {{ r_subscription }}"
+- name: Remove CSV
```
Comment (stencell) [inline ansible/roles/ocp4-workload-pipelines/tasks/remove_workload.yml]: formatting nit: add line above

Also, do you need to remove the CSV or will removing the subscription take care of this also?

Comment (wkulhanek) [reply ansible/roles/ocp4-workload-pipelines/tasks/remove_workload.yml]: Need to. Removing just the subscription leaves the CSV.

**Pull request merged**